pkg/controller/controller.go

```diff
@@ -242,6 +260,11 @@ func (c *Controller) sync(domain string) error {
 		if rdl.Code == 404 {
 			return c.cr.RemoveAthenzDomain(domain)
 		}
+		obj, e, _ := c.cr.GetCRByName(domain)
+		if e && obj.Status.Message == "" {
```

Comment:
Should we be updating the error status message every single time? Not just when it is empty?